firewalld/config.sls

```diff
@@ -26,6 +24,7 @@ config_firewalld:
     - require:
       - pkg: package_firewalld # make sure package is installed
       - file: directory_firewalld
-    - listen_in: 
-      - module: service_firewalld # restart service
-
+    - require_in:
+      - service: service_firewalld
+    - watch_in:
```

Comment:
why not keep the listen_in?

that way only after all the changes it will be reloaded, instead of everytime.